_aggregations/pipeline/bucket-selector.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: default
+title: Bucket selector
+parent: Pipeline aggregations
+nav_order: 30
+redirect_from:
+  - /query-dsl/aggregations/pipeline-agg#bucket_script-bucket_selector/
+  - /query-dsl/aggregations/pipeline/bucket-selector/
+---
+
+# Bucket selector aggregations
+
+The `bucket_selector` aggregation is a parent pipeline aggregation that evaluates a script to determine whether buckets returned by a `histogram` (or `date_histogram`) aggregation should be included in the final result. 
+
+Unlike pipeline aggregations that create new values, the `bucket_selector` aggregation acts as a filter, keeping or removing entire buckets based on the specified criteria. Use this aggregation to filter buckets based on the computed metrics of a bucket. 
+
+## Parameters
+
+The `bucket_selector` aggregation takes the following parameters.
+
+| Parameter             | Required/Optional | Data type       | Description |
+| :--                   | :--               |  :--            | :--         |
+| `buckets_path`        | Required          | Object          | A map of variable names to bucketed metrics that identify the metrics to be used in the script. The metrics must be numeric. See [Script variables]({{site.url}}{{site.baseurl}}/aggregations/pipeline/bucket-script#script-variables) as described in the `bucket_script` aggregation. |
```

Comment:
Last sentence: Instead of "as described in the `bucket_script` aggregation", should this link to the "Bucket script" section/page?

Reply:
It already links to the section, so I'll remove the "as described".